### PR TITLE
Whispers - start text under "whispered" not inline

### LINKED ISSUE #3651: Whispers - start text under "whispered" not inline
y2m25EvqTBGUj1Y_EjOfOA

### PR DESCRIPTION
**Pull request by anjandev**
closes #3651

### CODE REVIEW

**Comment by licaon-kter** [comment]
Wow, so simple? Thanks

**Comment by licaon-kter** [comment]
Any blockers?

**Comment by licaon-kter** [comment]
There's an UI refactor that blocks this?

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -254,8 +254,8 @@
     <string name="error_saving_avatar">Could not save avatar to disk</string>
     <string name="or_long_press_for_default">(Or long press to bring back default)</string>
     <string name="error_publish_avatar_no_server_support">Your server does not support the publication of avatars</string>
-    <string name="private_message">whispered</string>
-    <string name="private_message_to">to %s</string>
+    <string name="private_message">whispered\n</string>
```
Comment (mimi89999) [inline src/main/res/values/strings.xml:257]: Please don't do it like that. The trailing newline can be accidentally removed by translators. There are better ways to do that. Please see `src/main/java/eu/siacs/conversations/ui/adapter/MessageAdapter.java`

**Review by mimi89999 — CHANGES_REQUESTED** [review]

File: src/main/res/values/strings.xml (hunk 1 above)
Comment (iNPUTmice) [reply src/main/res/values/strings.xml:257]: I agree

**Comment by licaon-kter** [comment]
@mimi89999 so you're gonna PR?

File: src/main/res/values/strings.xml (hunk 1 above)
Comment (licaon-kter) [reply src/main/res/values/strings.xml:257]: @mimi89999 newlines and special chars are enforced by the translation portal, afaik

Comment (iNPUTmice) [reply src/main/res/values/strings.xml:257]: That's still not a good solution. The proper solution is to work with an additional TextView.

**Pull request closed without merging**